Source/WebGPU/WGSL/Parser.cpp

```diff
@@ -1154,4 +1160,28 @@ Result<AST::Value::Ref> Parser<Lexer>::parseLetValue()
     RETURN_NODE_UNIQUE_REF(LetValue, WTFMove(name), WTFMove(maybeType), WTFMove(initializer));
 }
 
+template<typename Lexer>
+Result<AST::Value::Ref> Parser<Lexer>::parseOverrideValueWithAttributes(AST::Attribute::List&& attributes)
```

Comment:
Same comment as in #10123. Here the initializer is also optional, but that should be fine as well.